code/game/objects/structures/mirror.dm

```diff
@@ -17,6 +17,7 @@
 		if(!AC)
 			AC = new(src, user)
 			AC.name = "SalonPro Nano-Mirror&trade;"
+			AC.flags = APPEARANCE_ALL_BODY

```

Comment:
Means that users of appropriately flagged species will be able to change their head accessory (and its colour), their markings (and their colour) and their body accessory at a mirror. Technology is wonderful.